Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -267,14 +267,24 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
     contextMenu->addSeparator();
   }
 
+  MemWatchEntry* const entry{MemWatchModel::getEntryFromIndex(index)};
+  if (!entry) {
```

Comment:
It kind of makes sense to me to also show these two actions if the right-clicked entry is a group. Thoughts?

Reply:
probably but then one would need to code it so that the newly added watch-goes inside the clicked-on group

Reply:
I thought that was the case already. But if it isn't, then it can be looked at at another time.